flink-python/pyflink/datastream/stream_execution_environment.py

```diff
@@ -35,6 +35,8 @@
 from pyflink.datastream.time_characteristic import TimeCharacteristic
 from pyflink.java_gateway import get_gateway
 from pyflink.serializers import PickleSerializer
+from pyflink.table.types import _from_java_type, _infer_schema_from_data, _create_converter,\
```

Comment:
Importing table types in StreamExecutionEnvironment might be confused logically since table API is based on StreamExecutionEnvironment.

Reply:
@shuiqiangchen Of course, I would like to extract these methods to util methods.